pyblish_bumpybox/plugins/validate_scene_version.py

```diff
@@ -6,8 +8,12 @@ class ValidateSceneVersion(pyblish.api.ContextPlugin):
 
     order = pyblish.api.ValidatorOrder
     label = "Scene Version"
+    optional = True
 
     def process(self, context):
 
-        msg = "Could not find a version number in the scene name."
-        assert context.has_data("version"), msg
+        name, ext = os.path.splitext(context.data("currentFile"))
+        msg = "Could not find a version number in the scene name. Please add "
+        msg += "v[number] to the scene name, for example: "
+        msg += "\"{0}_v001{1}\".".format(name, ext)
+        assert "version" in context.data, msg
```

Comment:
Why would the string creation method affect how pyblish lite renders it? It'll be the same string in memory when it's used won't it?

     failure_message = "Could not find a version number in the scene name. Please add " \
                       "v[number] to the scene name, for example: " \
                       "\"{0}_v001{1}\".".format(name, ext)

Unless we have a coding convention written down, then we should be using PEP8 throughout. This is what ftrack is moving over to and the rest of the world uses.